rush.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-  "rushVersion": "5.112.2",
```

Comment:
Rush version should also be updated in the GitHub workflows

Reply:
They're no longer in the CI workflow; rush version is set by the `rush.json` file.
The other workflows should only need any rush version to bootstrap the workflow on the build machine

Reply:
What about these ones https://github.com/search?q=repo%3AAzure%2Fcommunication-ui-library%205.112.2&type=code ?

Reply:
`The other workflows should only need any rush version to bootstrap the workflow on the build machin`
I'm a little bit worried that we would skip those files and they will stuck with the currrent version

Reply:
Maybe we can just use the latest version in those workflows instead of the pinned one

Reply:
Maybe we can just remove the pinned versions in the other workflows @Leah-Xia-Microsoft ?